src/async/deserializeAsync.ts

```diff
@@ -284,11 +284,19 @@ export function createTsonParseEventSource(opts: TsonAsyncOptions) {
 
 		signal?.addEventListener("abort", onAbort);
 
+		// eventSource.addEventListener("head", (e) => {
+		// 	controller.enqueue(JSON.parse(e.data));
+		// });
+
 		eventSource.onmessage = (msg) => {
 			// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
 			controller.enqueue(JSON.parse(msg.data));
 		};
 
+		eventSource.addEventListener("close", () => {
+			controller.close();
+		});
```

Comment:
I don't think this event exists 👀

Reply:
I could have called it 'dirtySocks' and it would have the same effect. EventSource is just an EventEmitter consumer, and SSE is essentially an HTTP EventEmitter with some nifty connection management features.